File: DSA/Rahul_Mankani_590014725/Day15_Question3.c
#include <stdio.h>

int main() 
{
    int arr[100]; 
    int n; 
    int temp; 
    printf("Enter the size: ");
    scanf("%d", &n);
    if (n <= 0 || n > 100) 
    {
        printf("Incorrect size! Enter from 1 to 100.\n");
        return 1;
    }
    printf("Numbers daalo: ");
    for (int i = 0; i < n; i++) 
    {
        scanf("%d", &arr[i]);
    }
    for (int i = 0; i < n - 1; i++) 
    {
        int swapped = 0; 
        for (int j = 0; j < n - i - 1; j++) 
        {
            if (arr[j] > arr[j + 1]) 
            {
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
                swapped = 1;
            }
        }
        if (swapped == 0) break;
    }
    printf("Sorted array: ");
    for (int i = 0; i < n; i++) 
    {
        printf("%d ", arr[i]);
    }
    printf("\n");
    return 0;
}

File: DSA/NiharikaChand_590017697/Day15Question3.java
import java.util.*;
class Day15Question3 
{

    public static void display(int arr[], int n)
    {
        int i; 
        for(i=0;i<n;i++)
        {
            System.out.print(arr[i] + "\t");
        }
        System.out.println();
    }

    public static void sort(int arr[], int n) 
    {
        int i, j, temp;
        for (i = 0; i < n - 1; i++) 
        {
            for (j = 0; j < n - i - 1; j++)
            {
                if (arr[j] > arr[j + 1]) 
                {
                    temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
    }

    public static void main(String arg[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of array: ");
        int n= sc.nextInt();
        int arr[]=new int[n]; int i;

        for(i=0; i<n; i++)
        {
            System.out.println("Enter an element: ");
            arr[i]=sc.nextInt();
        }

        sort(arr,n);
        System.out.println("Sorted array: ");
        display(arr, n);
    }
}
